web/pgadmin/static/js/helpers/ModalProvider.jsx

```diff
@@ -260,6 +260,9 @@ export const useModalStyles = makeStyles((theme) => ({
   iconButtonStyle: {
     marginLeft: 'auto',
     marginRight: '4px'
+  },
+  noteStyle: {
+    color: theme.otherVars.errorColor
```

Comment:
So we always show modal notes in error color? And why use errorColor at all? Its a note right?